src/settings.lris_red

```diff
@@ -0,0 +1,203 @@
+science check condition1 lampstat01=off&lampstat02=off&lampstat03=off&lampstat04=off&lampstat05=off&lampstat06=off&lampstat07=off&lampstat08=off&lampstat09=off&lampstat10=off&lampstat11=off&lampstat12=off
+science check condition2 hatch=open # Required for science
+science check condition3 exptime>29 # Arbitrary exptime limit
+
+### Standard Star frames
+standard idname OBJECT               # Header key value of science frame for header keyword keyword+idname
+standard number 1                    # Number of standard star frames to use
+standard canbe None                  # If there are frames that will be science ***in addition to other frame types***, include the other frame types here.
+standard check condition1 lampstat01=off&lampstat02=off&lampstat03=off&lampstat04=off&lampstat05=off&lampstat06=off&lampstat07=off&lampstat08=off&lampstat09=off&lampstat10=off&lampstat11=off&lampstat12=off
+standard check condition2 hatch=open # Required for standard
+standard match disperser ''             # Check the same decker as the science frame was used
+standard match dichroic ''             # Check the same decker as the science frame was used
+standard match binning ''             # Match the shape of standard and science frames
+#standard match naxis0 =0             # Match the shape of standard and science frames
+#standard match naxis1 =0             # Match the shape of standard and science frames
+
+### Bias/Dark frames
+bias idname DARK                    # Header key value of bias frame for header keyword keyword+idname
+bias number 5                       # Number of bias frames to use
+bias canbe None                     # If there are frames that will be bias ***in addition to other frame types***, include the other frame types here.
+bias check condition1 lampstat01=off&lampstat02=off&lampstat03=off&lampstat04=off&lampstat05=off&lampstat06=off&lampstat07=off&lampstat08=off&lampstat09=off&lampstat10=off&lampstat11=off&lampstat12=off
+bias check condition2 hatch=closed  # Required for bias
+bias check condition3 exptime<1     # Required for bias
+bias match binning ''             		# Match the shape of standard and science frames
+#bias match naxis0 =0                # Match the shape of bias and science frames
+#bias match naxis1 =0                # Match the shape of bias and science frames
+
+### Pixel Flat frames
+pixflat idname OBJECT               # Header key value of flat frame for header keyword keyword+idname
+pixflat number 5                    # Number of flat frames to use
+pixflat canbe None                  # If there are frames that will be pixflat ***in addition to other frame types***, include the other frame types here.
+pixflat lscomb False                # Combine a long and short flat
+#pixflat check condition1 lampstat09=on|lampstat10=on|lampstat11=on|lampstat12=on
+pixflat check condition1 hatch=open   # Required for blaze flats
+pixflat check condition2 exptime<30   # Avoid stars
```

Comment:
Hmmm... perhaps we should think about algorithms that can autodetect what type of frame we are dealing with. It should be fairly straightforward to tell the difference between a twilight flat and a science exposure... In this case, it might be better to not include this as a default, and instead anyone who takes a twilight flat will need to set it as such in their .red script